register/emails.py

```diff
@@ -17,3 +21,52 @@ def sendgrid_send(recipients, subject, substitutions, template_id):
     # Replace substitutions in sendgrid template
     mail.substitutions = substitutions
     mail.send()
+
+
+class MailListManager:
+    WINTER_17_LIST_ID = "876178"
+
+    def __init__(self):
+        self.sg = sendgrid.SendGridAPIClient(apikey=SENDGRID_API_KEY)
+
+    def add_recipient_to_list(self, recipient_id, list_id):
```

Comment:
It's not right now but I didn't want to rule out the option of being called from outside, once an application has a sendgrid_id saved we might as well call this method instead of the other one. I'll add the _ because right now it's not used from the outside but in the need arises in the future let's just remove it again and use this method.